processes/travel-policy.md

```diff
@@ -0,0 +1,160 @@
+essential. The primary credit card circuits are American Express, Visa and MasterCard.
+When you travel outside the European Union, we'll provide medical insurance to cover possible
+injuries that require medical treatment.
+
+#### Useful mobile Apps
+
+- **Wi-Fi Finder,** detects free Wi-fi networks.
+- **MetrO, Kinevia** is a free mobile guide to public transport systems worldwide. It provides the
+fastest route to your point of interest considering the day of the week and the time of the day.
+- **Airline companies official App,** allows through your confirmation or ticket number to download
+an e-ticket for Wallet or Pass2U Wallet. Also, you'll have handy information about your flight.
+- **Mytaxi:** is a Taxi app available in 9 countries within the European Union, Italy included.
+
+## Conferences
+
+### Choosing the right conference
+
+In the following document you can find a conference list that you and Nebulab may be interested in
+relating to:
+- self-improvement conferences recommended for topics that are tight to Nebulab interests and
+business;
+- marketing conferences, which are attractive for our marketing initiatives.
+[Conferences list](notion://www.notion.so/982ae7532a5b49298a89ea0ecd0b8d66)
+If you want to submit some not-listed event, you can suggest a conference in the document or the
+Slack channel #conferenze.
+Your mentor will follow you in the choice. Propose your conference schedule, respecting our yearly
+[budget limitations](notion://www.notion.so/982ae7532a5b49298a89ea0ecd0b8d66), and explain why you
+are interested in those events according to your growth path and be sure to [get the most out of
+your conferences.](https://github.com/nebulab/playbook/blob/master/personal-growth/conferences.md)
+
+### Yearly budget and days amount
+
+Every year we give you a budget and a maximum amount of conference attendance days. In the
+conference list, you can find days and budget associated with each event.
+**2019 team member conference budget**
+- maximum of eight days;
+- maximum 3.400,00€.
+
+### Team sponsorship
+
+If you're in the conference marketing team, we won't deduct the cost and the days for that
+conference from your yearly budget.
+
+### Conference speaker
+
+Being invited as a speaker is an honor as well as, pairing with our values, a great way to spread
+your knowledge. At the same time, it will give us much visibility. Therefore, we encourage to submit
+speech proposal to sponsored, listed or even non-listed conferences. If you're invited as a speaker,
+we won't deduct the cost and the days for that conference from your yearly budget.
+
+## **Team Retreat**
+
+We will organize one Team Retreat per year, lasting one week. We'll know each other in a better way,
+building stronger relationships and mutual trust and improving communication and cooperation.
+Of course, we will have much fun.
+We'll ask you for confirmation with a three-month notice.
```

Comment:
Just a TODO note. Here we can say much more, maybe in a dedicated page for team retreats. I think we can do this later though.

Reply:
Yes, I agree